test/blank.test.js

```diff
@@ -30,5 +30,11 @@ describe('#blank', () => {
         expect(wrapper.find('#parent')).to.be.blank()
       }).to.throw(`to be empty`)
     })
+
+    it('fails when actual is undefined', () => {
```

Comment:
Can you using the same phrasing as the other assertions:

```js
it('fails when the actual is undefined', () => {});
```

Same with the other examples 😄